### PR TITLE
✨ Add create and edit rubrics

### PR DESCRIPTION
**Pull request by Jeffchiucp**
* Add level names to the rubrics model
* changes to Rubric model... allowed name to be null, set a defaultValu…

### CODE REVIEW

**Comment by Jeffchiucp** [comment]
@tlambrou @alengel please also approve this backend changes review when you get a chance

**Comment by Jeffchiucp** [comment]
@anselb  let me know when you get a chance to review the changes on the server side #47

**Comment by Jeffchiucp** [comment]
@tlambrou    the server repro is set up to require 1 approval in order for me to merge it, since @tlambrou is thebackend lead who owns the github, we have to get review and approval in order to merge this PR #47

File: models/rubric.js [hunk 1]
```diff
@@ -2,14 +2,40 @@
...
-      type: DataTypes.STRING
+      type: DataTypes.STRING,
+      defaultValue: 'work',
+    },
+    iconColor: {
+      allowNull: true,
+      type: DataTypes.STRING,
+      defaultValue: 'rose',
+    },
+    description: DataTypes.TEXT,
```
Comment (anselb) [inline models/rubric.js:18]: Should it be possible for the description to have a null value? Or at least some sort of defaultValue? If a user doesn't enter anything for the description, what will happen?

**Review by anselb — COMMENTED** [review]
I had a question regarding the rubrics model, but as far as I can tell, everything else looks good.

File: models/rubric.js (hunk 1 above)
Comment (Jeffchiucp) [reply models/rubric.js:18]: In that case, it will be null, which is okay and handled by the sequelize orm,

**Comment by Jeffchiucp** [comment]
@anselb  let me know if you can approve this PR.

**Review by anselb — APPROVED** [review]

**Pull request merged**